From 745f0f5e6c099cf2fa7b92f4a804377817fc9318 Mon Sep 17 00:00:00 2001
From: Naushir Patuck <naush@raspberrypi.com>
Date: Thu, 18 Feb 2021 15:05:57 +0000
Subject: [PATCH 0567/2201] media: i2c: imx477: Fix crop height for 2028x1080
 mode

The crop height for this mode was set at 2600 lines, it should be 2160
lines instead.

Signed-off-by: Naushir Patuck <naush@raspberrypi.com>
---
 drivers/media/i2c/imx477.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/imx477.c b/drivers/media/i2c/imx477.c
index 0143e3129258..e3d164555b09 100644
--- a/drivers/media/i2c/imx477.c
+++ b/drivers/media/i2c/imx477.c
@@ -955,7 +955,7 @@ static const struct imx477_mode supported_modes_12bit[] = {
 			.left = IMX477_PIXEL_ARRAY_LEFT,
 			.top = IMX477_PIXEL_ARRAY_TOP + 440,
 			.width = 4056,
-			.height = 2600,
+			.height = 2160,
 		},
 		.timeperframe_min = {
 			.numerator = 100,
-- 
2.17.1

